### PR TITLE
adding docker guide

### PR DESCRIPTION
**Pull request by danielcbright**
This is a quick guide on using bash aliases to run multiple versions of Chef DK/Workstation on a local machine, this is for Mac right now, Windows soon

### CODE REVIEW

File: examples/docker-ops/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      image="chef/chefworkstation:stable"
+      local args=("$@")
+      func_chef "$image" "${args[@]}"
+    }
+    ```
+1. Source the profile, on my Mac, using zshrc, I type `source ~/.zshrc`
+1. Run commands like this `chef_<version> [chef|knife|kitchen] arg1 arg2 etc....`:
+    ```shell
+    # Example - knife node list, ChefDK 3.12.10
+    > chef_14 knife node list
```
Comment (devopsdina) [inline examples/docker-ops/README.md:88]: Should this be `chefdk_3.12 knife node list`?  In your comment it has chefDK 3.12.10, but in the command it has chef_14

Comment (devopsdina) [reply examples/docker-ops/README.md:88]: kind of confusing IMO that we are calling chefdk 3.12.10, chef_14

Comment (danielcbright) [reply examples/docker-ops/README.md:88]: yeah it's a good point, I was trying to tie the dk/workstation to the version of Chef Client it was used for, if it makes more sense to alias the versions of the dk/workstation I can do that as well, maybe have multiple names for the alias (as a matter of fact, I'll do that and update the PR right now :) )

Comment (devopsdina) [reply examples/docker-ops/README.md:97]: Nice!  The other thought I had was that we want to ensure users understand they can run whatever version of chef-client, no matter what dk or workstation version they are using.  Sometimes customers think these two items are tied together.  More food for thought vs. action item.

Comment (danielcbright) [reply examples/docker-ops/README.md:97]: yeah good point, I guess the idea here is "if you are developing for "x-version" of chef, use this image", that way users can switch between different versions when needed (like when supporting older versions of legacy cookbooks for whatever reason, while also developing for newer versions)

**Pull request merged**